samples/retained-instance-store/src/main/java/com/badoo/ribs/samples/retained_instance_store/rib/builder/RetainedInstanceBuilder.kt

```diff
@@ -0,0 +1,35 @@
+package com.badoo.ribs.samples.retained_instance_store.rib.builder
+
+import com.badoo.ribs.builder.SimpleBuilder
+import com.badoo.ribs.core.modality.BuildParams
+import com.badoo.ribs.samples.retained_instance_store.rib.*
+import com.badoo.ribs.samples.retained_instance_store.rib.RetainedInstanceNode
+import com.badoo.ribs.samples.retained_instance_store.rib.RetainedInstancePresenterImpl
+import com.badoo.ribs.samples.retained_instance_store.utils.ScreenOrientationController
+import com.badoo.ribs.store.RetainedInstanceStore
+import com.badoo.ribs.store.get
+
+
+class RetainedInstanceBuilder(
+        private val dependency: RetainedInstanceRib.Dependency
+) : SimpleBuilder<RetainedInstanceRib>() {
+
+    override fun build(buildParams: BuildParams<Nothing?>): RetainedInstanceRib {
+
+        val retainedPresenter = RetainedInstanceStore.get(
```

Comment:
Don't retain the presenter this way. Retain only the object which does the long running operation itself.

Reply:
Done